application/models/TemplateConfiguration.php

```diff
@@ -1593,12 +1593,18 @@ public function setOptionKeysToInherit()
      */
     public function sanitizeImagePathsOnJson($attribute, $params)
     {
+        $excludedOptions = [
+            'cssframework'
```

Comment:
You add it here : how add other ?

**Need** a global solution. Something than can be done by config.xml or HTML options.js

Reply:
It could be a global config setting?
I would say to add a new ticket so we can discuss it there.

Reply:
Again a ,new ticket …

Hacking is really more the quickest to improve LS

Reply:
Hacking is the quickest way to loose track of changes, arguments and decisions. Also not very helpfull for coordinating in a team.

Reply:
I'm OK, but i have fix since month now … and i have to give proof of reach of my sentence.

Reply:
> I'm OK, but i have fix since month now … and i have to give proof of reach of my sentence.

which fix?

Reply:
Still have some, don't remind of the one related here.